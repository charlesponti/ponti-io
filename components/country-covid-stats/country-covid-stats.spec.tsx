import renderWithProviders from "@/tests/test.utils";
import { describe, expect, test } from "vitest";

import CountryCovidStats from ".";

describe("CountryCovidStats", () => {
	test("should render global states", async () => {
		const { container } = renderWithProviders(
			<CountryCovidStats countryCode="global" />,
		);
		expect(container).toMatchSnapshot();
	});

	test("should render US data", () => {
		const { container } = renderWithProviders(
			<CountryCovidStats countryCode="US" />,
		);
		expect(container).toMatchSnapshot();
	});
});
