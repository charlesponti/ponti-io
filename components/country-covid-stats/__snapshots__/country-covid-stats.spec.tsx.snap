// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CountryCovidStats > should render US data 1`] = `<div />`;

exports[`CountryCovidStats > should render global states 1`] = `<div />`;
